### PR TITLE
Added apm_enabled

### PR DESCRIPTION
**Pull request by jjasghar**
This will enable apm_enabled to be part of this cookbook.

Signed-off-by: JJ Asghar <jj@chef.io>

### CODE REVIEW

File: templates/default/datadog.conf.erb [hunk 1]
```diff
@@ -37,6 +37,10 @@ proxy_user: <%= node['datadog']['web_proxy']['user'] %>
 proxy_password: <%= node['datadog']['web_proxy']['password'] %>
 <% end -%>
 
+<% if node['datadog']['apm_enable'] == true %>
+  apm_enable: true
```
Comment (robbkidd) [inline templates/default/datadog.conf.erb]: This should be `apm_enabled` with a `d`. ( [code](https://github.com/DataDog/datadog-trace-agent/blob/dc580a0cd378bf0723e5af099605cf9d410a921f/config/agent.go#L196), [docs](https://app.datadoghq.com/trace/install) )

Comment (jjasghar) [inline templates/default/datadog.conf.erb]: ❤️

**Comment by robbkidd** [comment]
👍  Without this, I am faced with declaring `apm_enabled: true` as either a `legacy_integrations` or an `extra_config`

**Comment by mlcooper** [comment]
Looks like a this will be taken care of with #401 which has some additional granular settings for APM and recipes for the require python/ruby packages

**Comment by jjasghar** [comment]
Sounds good, thanks!

**Pull request closed without merging**